graph.py

```diff
@@ -115,85 +114,94 @@ def incidence(self) -> List["Edge"]:
 
     @property
     def neighbours(self) -> ["Vertex"]:
```

Comment:
We moeten nog even bedenken hoe we dit doen nu een vertex bij meerdere graven hoort. Hier 'graaf' als param meegeven?

Reply:
Had ik al geprobeerd: is een bitch om te maken. Niet doen dus.

Wat gaat er mis met de huidige definitie?

Reply:
Dat de neighbours van een vertex in verschillende graven kunnen liggen.
En wat dacht je van 
```
def neighbours(self, graph: Graph) -> ["Vertex"]:
    return [v in self._incidence.keys() for v.in_graph(graph)] 
```

Reply:
Ja, ze kunnen uit verschillende grafen komen. Wederom een design flaw van wat we hebben gekregen. En je fix had ik ook ongeveer zo bedacht, dat is ook niet het probleem. Het probleem is dat dit echt ellendig om overal te refactoren, want `Vertex#neighbours` wordt op nogal wat plekken gebruikt waar het bijbehorende graafobject niet beschikbaar is.